src/openMVG/features/cctag/CCTAG_describer.hpp

```diff
@@ -41,15 +45,26 @@ class CCTAG_Image_describer : public Image_describer
   template<class Archive>
   void serialize( Archive & ar )
   {
-    //ar(
-     //cereal::make_nvp("params", _params),
-     //cereal::make_nvp("bOrientation", _bOrientation));
+    ar(
+     cereal::make_nvp("cannyThrLow", _params._cannyThrLow),
+     cereal::make_nvp("cannyThrHigh", _params._cannyThrHigh));
   }
 
+  struct CCTagParameters
+  {
+    CCTagParameters(size_t nRings);
+    ~CCTagParameters();
+
+    float _cannyThrLow;
+    float _cannyThrHigh;
+
+    bool setPreset(EDESCRIBER_PRESET preset);
```

Comment:
maybe move it up so that we have the members all together and the methods/constructors/destructors all together